Formula/gr-osmosdr.rb

```diff
@@ -0,0 +1,19 @@
+class GrOsmosdr < Formula
+  desc "Osmocom Gnu Radio Blocks"
+  homepage "https://osmocom.org/projects/sdr/wiki/GrOsmoSDR"
+  url "https://cgit.osmocom.org/gr-osmosdr/snapshot/gr-osmosdr-0.1.4.tar.xz"
+  sha256 "1945d0d98fd4b600cb082970267ec2041528f13150422419cbd7febe2b622721"
+  # head "git://git.osmocom.org/gr-osmosdr"
+  head "git://github.com/mikemorris/gr-osmosdr.git"
```

Comment:
Why did you provide your fork of gr-osmosdr as repository for the head version instead of the official one (or its GitHub mirror)?

Reply:
Remove head

Reply:
I added my fork because of a small patch required for it to compile with clang-9 (default on macOS 10.13.2, haven't tested with other compilers) - https://github.com/mikemorris/gr-osmosdr/commit/ae9bd438c31d5c519cd0874027952f11bdba9cf4

Is there a way to move this patch into the Homebrew formula to keep the official repository as `head`?

Reply:
Figured out how to add the patch in formula, hopefully it will be accepted upstream though.